definitions/region/african_subregions.yaml

```
@@ -21,3 +21,103 @@
         Zambia, Zimbabwe
 - Africa:
   - Africa
+
+# Nigerian zones and states defined according to:
+#  - https://maps-nigeria.com/map-of-nigeria-showing-the-six-geopolitical-zones
+#  - https://en.wikipedia.org/wiki/States_of_Nigeria
+
+- Nigeria Zones:
+  - North Central:
```

In analogy I'd also prefix this with `Nigeria Zones|`:
```suggestion
  - Nigeria Zones|North Central:
```
This would improve clarity as for the published output the hierarchy name `Nigeria Zones` in this case is not written but only the region name (`North Central`).